Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "picorv32"
-version = "0.1.3"
+version = "0.1.4"
```

Comment:
I'd suggest 0.2.0.
Just in case it breaks something, at least people who use "0.1" will not be affected.

Reply:
Updated to 0.2.0.